### DISCUSSION TITLE
'gcs_schema_object' not picked up when using "jagged_rows: True'

### DISCUSSION THREAD

**Discussion by rasalt** [apache/airflow#18358] [Q&A]
Goal:
I Want to allow jagged rows and specify a json file with the schema definition for the BQ destination table.
This works as expected if I use the "schema_fields" argument but does not when I use the gcs_schema_object argument.
Please advise.

**Apache Airflow version**:

**Kubernetes version (if you are using kubernetes)** (use `kubectl version`):

**Environment**:

- **Cloud provider or hardware configuration**: GCP
- **OS** (e.g. from /etc/os-release):
- **Kernel** (e.g. `uname -a`):
- **Install tools**:
- **Others**:

**What happened**:

**What you expected to happen**:

**How to reproduce it**:

**Anything else we need to know**:

**Comment by boring-cyborg** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!
